content/docs/tasks/security/auth-sds/index.md

```diff
@@ -0,0 +1,77 @@
+
+Workload requests key/certificate from SDS server(node agent, which runs as per-node daemon set) using k8s service account JWT through
+SDS API, node agent generates private key and send CSR request to Citadel, Citadel verifies the JWT and signs the certificate,
+the key/certificate will eventually be sent back to workload sidecar through SDS server(node agent).
+Since key/certificate which are generated through SDS are stored in workload sidecar's memory, it's more secure than file mount;
+also the workload doesn't need to restart to pick up the certificate change when rotation happens.
+
+## Before you begin
+
+* Set up Istio by following the instructions using [Helm](/docs/setup/kubernetes/helm-install/) with SDS setup and global mutual TLS enabled:
+
+    {{< text bash >}}
+    $ cat install/kubernetes/namespace.yaml > istio-auth-sds.yaml
+    $ cat install/kubernetes/helm/istio-init/files/crd-* >> istio-auth-sds.yaml
+    $ helm dep update --skip-refresh install/kubernetes/helm/istio
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml >> istio-auth-sds.yaml
+    $ kubectl create -f istio-auth-sds.yaml
+    {{< /text >}}
+
+## service-to-service mutual TLS using key/certificate provisioned through SDS
+
+Follow task in [authentication policy](/docs/tasks/security/authn-policy/) to setup test services.
+
+{{< text bash >}}
+$ kubectl create ns foo
+$ kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml) -n foo
+$ kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml) -n foo
+$ ubectl create ns bar
+$ kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml) -n bar
+$ kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml) -n bar
+{{< /text >}}
+
+Verify all mutual TLS requests succeed.
+
+{{< text bash >}}
+$ for from in "foo" "bar"; do for to in "foo" "bar"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl "http://httpbin.${to}:8000/ip" -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
+sleep.foo to httpbin.foo: 200
+sleep.foo to httpbin.bar: 200
+sleep.bar to httpbin.foo: 200
+sleep.bar to httpbin.bar: 200
+{{< /text >}}
+
+## Verifying no secret-mount file is generated
+
+As mentioned, instead of using secret-mount file, key/certificate provisioned by SDS flow are stored in workload sidecar's memory,
+which is more secure and avoid traffic interruption.
+Let's verify no secret-mount file is generated by remote accessing to workload sidecar container that we just deployed.
+
+{{< text bash >}}
+$ kubectl exec -it $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c istio-proxy -n foo  -- /bin/bash
+{{< /text >}}
+
+As you can see there is no secret file mounted at `/etc/certs` folder.
+
+## Cleanup
+
```

Comment:
You need one sentence to introduce the following text block.

Reply:
added